molecule/api.py

```diff
@@ -46,3 +46,22 @@ def drivers(as_dict=False, config=None):
         return plugins
     else:
         return sorted(list(plugins.keys()))
+
+
+@lru_cache()
+def verifiers():
+    plugins = {}
+    pm = pluggy.PluginManager("molecule.verifier")
+    try:
+        pm.load_setuptools_entrypoints("molecule.verifier")
+    except Exception:
+        # These are not fatal because a broken verifier should not make the entire
+        # tool unusable.
+        LOG.error("Failed to load verifier entry point %s", traceback.format_exc())
+    for p in pm.get_plugins():
+        try:
+            plugins[pm.get_name(p)] = p(None)
```

Comment:
`p(None)` looks strange! Is there something we can do about that instead ending up `p()`?